tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -455,10 +455,9 @@ Select subject "${SUBJECT_2_NAME}" in table tool
     user checks previous table tool step contains    2    Subject    ${SUBJECT_2_NAME}
 
 Select locations in table tool
-    # LH: Not using the 'user opens details dropdown' keyword here as
+    # don't use the 'user opens details dropdown' keyword here as
```

Comment:
I think you can probably just remove this whole coment now in my opinion.  I don't think people would question it if they saw something getting selected with its testid, so I don't think we need to maintain a explanation :)